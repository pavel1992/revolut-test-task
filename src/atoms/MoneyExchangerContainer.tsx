import styled from 'styled-components';

import { SMALL_DESKTOP_BREAKPOINT, TABLET_BREAKPOINT } from '../styleConstants/mediaConstants';

export const MoneyExchangerContainer = styled.div`
  width: 1160px;
  display: flex;
  flex-direction: column;

  @media(max-width: ${SMALL_DESKTOP_BREAKPOINT}) {
    width: 100%;
    padding: 0 60px;
  }

  @media(max-width: ${TABLET_BREAKPOINT}) {
    padding: 0;
  }
`;
