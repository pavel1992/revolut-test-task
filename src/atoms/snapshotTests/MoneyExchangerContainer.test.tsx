import 'jest-styled-components';
import * as React from 'react';
import renderer from 'react-test-renderer';

import { MoneyExchangerContainer } from '../MoneyExchangerContainer';

test('renders MoneyExchangerContainer correctly', () => {
  const render = renderer
    .create(<MoneyExchangerContainer>111</MoneyExchangerContainer>)
    .toJSON();
  expect(render).toMatchSnapshot();
});
