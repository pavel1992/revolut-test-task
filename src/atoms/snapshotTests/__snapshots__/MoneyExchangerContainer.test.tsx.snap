// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MoneyExchangerContainer correctly 1`] = `
.c0 {
  width: 1160px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

@media (max-width:1280px) {
  .c0 {
    width: 100%;
    padding: 0 60px;
  }
}

@media (max-width:768px) {
  .c0 {
    padding: 0;
  }
}

<div
  className="c0"
>
  111
</div>
`;
